src/clients/UBAClient/index.ts

```diff
@@ -1,4 +1,5 @@
 export { UBAActionType } from "../../UBAFeeCalculator/UBAFeeTypes";
 export { BalancingFeeReturnType as UBABalancingFee, SystemFeeResult as UBASystemFee } from "./UBAClientTypes";
+export { refundRequestIsValid } from "./UBAClientUtilities";
```

Comment:
We may want to export everything on the off-chance it's needed

Reply:
```suggestion
export * from "./UBAClientUtilities";
```